glide.yaml

```diff
@@ -115,3 +115,4 @@ import:
   version: 5a004441f897722c627870a981d02b29924215fa
 - package: github.com/mitchellh/hashstructure
   version: b098c52ef6beab8cd82bc4a32422cf54b890e8fa
+- package: github.com/mattn/go-zglob
```

Comment:
Yep not sure how that was missed. I'm going to revisit this anyways since `zlog` is clearly not a drop in replacement